system/runners/BaseRunner.cfc

```diff
@@ -195,11 +195,14 @@ component {
 		required testResults,
 		required targetMD
 	){
-		var testBundles = arguments.testResults.getTestBundles();
+		var pathPatternMatcher = new testbox.system.util.PathPatternMatcher();
+		var testBundles        = arguments.testResults.getTestBundles();
 
-		// verify we have some?
 		if ( arrayLen( testBundles ) ) {
-			return ( arrayFindNoCase( testBundles, arguments.bundlePath ) ? true : false );
+			return pathPatternMatcher.matchPatterns(
+				testBundles.map( ( bundle ) => replace( bundle, ".", "/", "all" ) ),
+				replace( arguments.bundlePath, ".", "/", "all" )
```

Comment:
matchPatterns actually works as-is without doing the replace operations, but it doesn't explicitly support dot paths.  I think using replace will prevent any future regressions from changes in PathPatternMatcher that don't account for dot paths.